docs/internals/deprecation.txt

```diff
@@ -71,6 +71,8 @@ details on these changes.
 * Support for passing positional arguments to ``Model.save()`` and
   ``Model.asave()`` will be removed.
 
+* Having URL converters with the same name is deprecated
```

Comment:
You cannot have multiple converters with the same name so this is confusing. Also, this note should end with `will be removed` or contain `will no longer` etc.

Reply:
Maybe:
```
* ``django.urls.register_converter()`` will no longer allow overriding existing
  converters.
```